src/methods/defining_methods/medium/q5/InverseMethod.java

```diff
@@ -0,0 +1,31 @@
+package methods.defining_methods.medium.q5;
+
+/*
+Write a Java Program that takes a boolean variable and inverses the value of the variable
```

Comment:
Change question text to:
Write a Java Program that takes a boolean variable and inverts the value (if the value is true, make it false, and vice. versa.). Write a Java Method that accepts the boolean variable and returns a boolean value.